core_lib/cache/cache_decorator.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from datetime import timedelta, datetime, timezone
 from functools import wraps
-from typing import Union
+from typing import Union, Any

```

Comment:
Who ius using `Union` and `Any`?

Reply:
removed `Any`, `Union` is being used in the `init` for `expire: Union[timedelta, str] = None`